stream-video-android-core/api/stream-video-android-core.api

```diff
@@ -1,6 +1,5 @@
 public final class io/getstream/video/android/BuildConfig {
 	public static final field BUILD_TYPE Ljava/lang/String;
-	public static final field CORE_VIDEO_DOMAIN Ljava/lang/String;
```

Comment:
This is an unrelated change but this entry reappeared here in some previous commit by accident. So I will use the opportunity to also remove it (the CI API check would complain otherwise).